frontend/app/routes/$lang+/_public+/apply+/$id+/review-information.tsx

```diff
@@ -6,6 +6,7 @@ import { useFetcher, useLoaderData } from '@remix-run/react';
 
 import { faSpinner, faX } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { randomBytes } from 'crypto';
```

Comment:
```suggestion
import { randomBytes } from 'node:crypto';
```

Reference:
- <https://nodejs.org/docs/latest-v20.x/api/crypto.html#cryptorandombytessize-callback> (switch it to ESM syntax and look at example)